docs/swarm-api.md

```diff
@@ -24,71 +27,157 @@ POST "/images/create" : "docker import" flow not implement
 
 ## Endpoints which behave differently
 
-* `GET "/containers/{name:.*}/json"`: New field `Node` added:
+<table>
```

Comment:
@nishanttotla if you can make a list of things that are not supported, I'm happy to create a (follow-up) PR. I think @moxiegirl will do a documentation update today, so if this LGTY, it can be cherry-picked into the docs-branch so that it can be updated today (hopefully)